pkg/operator/k8sutil/deployment.go

```diff
@@ -144,6 +144,25 @@ func WaitForDeploymentToStart(ctx context.Context, clusterdContext *clusterd.Con
 	return fmt.Errorf("gave up waiting for deployment %q to update", deployment.Name)
 }
 
+func WaitForPodsWithLabelToRun(ctx context.Context, clientset kubernetes.Interface, namespace, label string) error {
+	// wait for the pods to run for a maximum of 100 sec
+	sleepTime := 5
+	attempts := 20
```

Comment:
Previously, the helper was waiting a total of up to 300s. How about we increase the waits so it's the same timeout?
```suggestion
	attempts := 60
```

Reply:
changed